docs/cookbook/syncing/full-scan-vs-sync.md

```diff
@@ -1,51 +1,66 @@
 # Full Scan vs Sync
 
-The BDK libraries rely on two syncing concepts we call _**Full Scan**_ and _**Sync**_. This page explains the difference between the two and recommendations on when to use each.
+Two of the four blockchain client libraries maintained by BDK (Electrum and Esplora) use what we refer to as _SPK-based syncing_ (as opposed to _block-by-block_ syncing). This SPK-based syncing relies on two concepts we call _**Full Scan**_ and _**Sync**_. This page explains the difference between the two, the tradeoffs made when choosing each, and recommendations on when to use them.
 
 ## Full Scan
 
-A _full scan_ is an operation that aims to build a complete picture of a wallet's UTXOs from scratch, with no prior data or knowledge about the wallet required. It involves querying the blockchain data client for ScriptPubKeys that the wallet owns, typically at a number of indices on two keychains, an external keychain and an internal one (often called a _change_ keychain). Because the wallet doesn't know which scripts/addresses have been given out and potentially have funds associated with them, it needs to query the blockchain client for a group of addresses, see if any of those have balances, and keep asking for more until a number of addresses in a row for each keychain are unused (the exact number is configurable and is what's typically known as a _stop gap_). The operation also returns a new chain tip the wallet uses to update its [`LocalChain`](https://docs.rs/bdk_chain/latest/bdk_chain/local_chain/struct.LocalChain.html).
+A _full scan_ is an operation that aims to build a complete picture of a wallet's UTXOs from scratch, with no prior data or knowledge about the wallet. It involves querying the blockchain data client for ScriptPubKeys that the wallet owns, typically at a number of indices on the wallet's keychains. 
 
-A full scan is only needed in cases where the wallet is unknown and is not loaded from persistence. In almost all other instances, a sync should be performed.
+Because the wallet doesn't know which scripts/addresses have been given out and potentially have funds associated with them, it needs to query the blockchain client for a group of addresses, see if any of those have balances, and keep asking for more until a number of addresses in a row for each keychain are unused (the exact number is configurable and is what's typically known as a _stop gap_). The operation also returns a new chain tip the wallet uses to update its [`LocalChain`](https://docs.rs/bdk_chain/latest/bdk_chain/local_chain/struct.LocalChain.html).
+
+A full scan is needed in cases where the wallet is unknown and is not loaded from persistence.
```

Comment:
I know you mention that full scan is needed when using with multiple software wallets, but I think it should also be mentioned here. Or I think just removing this line all together would work as well.

Reply:
Good point. I'll push a fix.